config/v3_1_experimental/translate/translate.go

```diff
@@ -33,9 +33,61 @@ func translateFilesystem(old old_types.Filesystem) (ret types.Filesystem) {
 	return
 }
 
+func translateConfigReference(old old_types.ConfigReference) (ret types.ConfigReference) {
+	// use a new translator so we don't recurse infintitely
```

Comment:
Minor: typo... though I see now it's just copy pasted from above. Meh, up to you if you want to fix this :)

Reply:
done :)